conformance/conformance.proto

```diff
@@ -70,8 +71,14 @@ enum TestCategory {
   JSON_IGNORE_UNKNOWN_PARSING_TEST = 3;
   // Test jspb wire format. Google internal only. Opensource testees just skip it.
   JSPB_TEST = 4;
+  // Test text format. Google internal only currently. Opensource won't have any
```

Comment:
This does not appear to be correct given the Java conformance test is trying to support them.  So it would appear some tests will be in opensource soon.

Reply:
Done fixing by changing to "for now".

Reply:
Probably mention only c++, java and python currently need to care about it.

Reply:
Done